src/main/java/net/snowflake/client/jdbc/SnowflakeConnectString.java

```diff
@@ -154,7 +154,7 @@ public static SnowflakeConnectString parse(String url, Properties info) {
 
       return new SnowflakeConnectString(scheme, host, port, parameters, account);
     } catch (Exception ex) {
-      logger.warn("Exception thrown while parsing Snowflake connect string", ex);
+      logger.info("Exception thrown while parsing Snowflake connect string", ex);
```

Comment:
Using info instead of debug because of the unexpected exception.

Reply:
@sfc-gh-wshangguan If it's an unexpected exception, yet, it's not critical, then that's the definition of a warning. If the log messages serve no purpose, then we should consider removing them altogether. We don't really want to see log messages saying "Exception" at INFO level either.

Reply:
Fair point. I'll keep this one at warning level.